src/main/java/com/jfb/tickets/service/TicketService.java

```diff
@@ -19,33 +20,32 @@ public class TicketService {
     @Value("classpath:ticketData.txt")
     Resource resource;
 
-    private final TicketDAO ticketDAO;
+    private final TicketRepository ticketRepository;
 
-    public TicketService(TicketDAO ticketDAO) {
-        this.ticketDAO = ticketDAO;
+    public TicketService(TicketRepository ticketRepository) {
+        this.ticketRepository = ticketRepository;
     }
```

Comment:
instead of creating constructors explicitly, you can you Lombok annotations `@RequiredArgsConstructor` and `@AllArgsConstructor`